virtualbox/README.md

```diff
@@ -0,0 +1,58 @@
+# VirtualBox scripts
+
+**This folder contains several scripts related to enhance building, exporting, and using FLARE-VM in VirtualBox.**
+
+## Clean up snapshots
+
+Its is not possible to select and delete several snapshots in VirtualBox, making cleaning up your virtual machine (VM) manually after having creating a lot snapshots time consuming and tedious (possible errors when deleting several snapshots simultaneously).
```

Comment:
Minor typo - "Its is not possible to select and delete", first word should just be "It"